### PR TITLE
Prerelease v4.2.5

### LINKED ISSUE #202: Pre-Release v4.2.5 Kubernetes
- [x] Bump PR from release branch to 4.2
  - [x] Test 4.2 branch

### PR DESCRIPTION
**Pull request by vcerenu**
This PR closes #202

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1,2 +1,2 @@
-VERSION="4.2.4"
-REVISION="40219"
+VERSION="5"
```
Comment (lastranikos) [inline VERSION:1]: It should be 4.2.5

**Review by lastranikos — COMMENTED** [review]
Fix version in VERSION

File: VERSION (hunk 1 above)
Comment (alberpilot) [reply VERSION:1]: Agree

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the @lastranikos requested changes

**Review by teddytpc1 — APPROVED** [review]
LGTM

**Review by alberpilot — APPROVED** [review]
LGTM

**Pull request merged**